kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -66,6 +65,9 @@
   public static final String AVRO_CODEC_CONFIG = "avro.codec";
   public static final String AVRO_CODEC_DEFAULT = "null";
 
+  public static final String S3_RETRY_CONFIG = "s3.part.retry";
```

Comment:
I wouldn't even add this option, I think retrying should just be the default. If I was going to add an option at all, it'd be the number of retries.